app/views/referee_interface/reference/feedback.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :title, t('page_titles.application') %>
+
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+
+  <%= form_with model: @reference_form, url: referee_interface_submit_feedback_path, method: :patch do |f| %>
```

Comment:
I'm surprised the token is automatically passed into this URL, didn't know Rails did that!

Reply:
you made me doubt of it :D, I think it is working,  I'll double check

Reply:
anyway, now it uses a query string `?token=XXXXXX`